nano/node/nodeconfig.hpp

```diff
@@ -69,14 +71,18 @@ class node_flags
 {
 public:
 	node_flags ();
-	bool disable_backup;
-	bool disable_lazy_bootstrap;
-	bool disable_legacy_bootstrap;
-	bool disable_wallet_bootstrap;
-	bool disable_bootstrap_listener;
-	bool disable_unchecked_cleanup;
-	bool disable_unchecked_drop;
-	bool fast_bootstrap;
-	size_t sideband_batch_size;
+	node_flags (boost::program_options::variables_map const &);
```

Comment:
`boost::program_options::variables_map` is quite a high level construct so I don't think `node_flags` should have a dependency on it. I think you can just move the logic contained in the constructor somewhere higher up (entry.cpp) and set it similarly as before.

Reply:
Right